src/screens/Settings/About/AppVersionRow.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import VersionNumber from "react-native-version-number";
+import { Trans } from "react-i18next";
+import { Text } from "@ledgerhq/native-ui";
+import cleanBuildVersion from "../../../logic/cleanBuildVersion";
+import SettingsRow from "../../../components/SettingsRow";
+
+function AppVersionRow() {
+  const { appVersion, buildVersion } = VersionNumber;
+  const version = `${appVersion || ""} (${cleanBuildVersion(buildVersion) ||
```

Comment:
Since we are already using the same thing on the `useExportLogs` hook, consider extracting the logic of the clean app version into something reusable for both cases

Reply:
Good idea, done